language/syntaxes/dfdl.tmLanguage.json

```diff
@@ -0,0 +1,387 @@
+{
+	"information_for_contributors": [
+		"This file has been converted from https://github.com/atom/language-xml/blob/master/grammars/xml.cson",
+		"If you want to provide a fix or improvement, please create a pull request against the original repository.",
+		"Once accepted there, we are happy to receive an update request."
+	],
+	"version": "https://github.com/atom/language-xml/commit/7bc75dfe779ad5b35d9bf4013d9181864358cb49",
```

Comment:
According to these links, this file is no ALv2 license. We need to track down the original license (it seems this file has been copied multiple times?) and ensure it works with ALv2 and mention it in the LICENSE file.

Reply:
@stevedlawrence I found the license to be

```
Copyright (c) 2014 GitHub Inc.

Permission is hereby granted, free of charge, to any person obtaining
a copy of this software and associated documentation files (the
"Software"), to deal in the Software without restriction, including
without limitation the rights to use, copy, modify, merge, publish,
distribute, sublicense, and/or sell copies of the Software, and to
permit persons to whom the Software is furnished to do so, subject to
the following conditions:

The above copyright notice and this permission notice shall be
included in all copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.

--------------------------------------------------------------------

This package was derived from a TextMate bundle located at
https://github.com/textmate/xml.tmbundle and distributed under the following
license, located in `README.mdown`:

Permission to copy, use, modify, sell and distribute this
software is granted. This software is provided "as is" without
express or implied warranty, and with no claim as to its
suitability for any purpose.
```

I am unsure of what kind of license it is, do you know?

Reply:
The part above the line is MIT. The part below which comes from the original work appears to just be a custom license, but is clearly very permissive and reads much like MIT but just less formal (and without the requirement to keep the copyright notice). Definitely compatible with ALv2, but needs to be included in LICENSE.

Reply:
Okay I will add that thank you.

Also with your issue downloading the omega-edit server, you have pulled down the latest code right their was an update to make sure it ran properly. But you can test running `yarn run omega-edit-download` to see if it works

Reply:
I'm building from commit 32eace89d73adc9f2f94feb97e4d7da1717cbf23, which I think is the latest.

> But you can test running yarn run omega-edit-download to see if it works

That does work. Is the difference that the "compile" script is just this

`"tsc -p ./ && node out/omega_edit/download.js`

The oega-edit-download script is this:

`"tsc -p ./ && node -e \"require('./out/omega_edit/download.js').downloadServer()\""`

I'm not sure why those two are different, or why we need the download in both places.

Reply:
Oh yeah sorry that was my fault I forgot to update the compile to remove that `node out/omega_edit/download.js` and run `yarn run omega-edit-download` instead. So that will be used up with the move to using sha512 instead of md5, adding this license to both LICENSE files, updating the MIT license text variable here soon.